CharlieBackend.Api/Controllers/StudentsController.cs

```diff
@@ -100,6 +100,22 @@ public async Task<ActionResult<IList<StudentDto>>> GetAllActiveStudents()
             return Ok(studentsModels);
         }
 
+        /// <summary>
```

Comment:
Why these changes are here?
I thought this branch is for unit-tests task, not for documentation and below changes in services and repository.
Please, separate it.

Reply:
Also, please, add description for your PR.

Reply:
Some changes are taken from another PR. I merged this branch from my earlier PR. This PR is about the last 3 commits